x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -377,9 +380,12 @@ func GetCalcSetNetworkRegrets(args GetCalcSetNetworkRegretsArgs) error {
 		}
 	}
 
-	// Recalculate topic initial regret
-	if len(workersRegrets) > 0 {
-		updatedTopicInitialRegret, err := CalcTopicInitialRegret(
+	// Check if we have enough workers with enough experience to calculate the topic initial regret
```

Comment:
You changed up the logic a lot relative to what I last reviewed, is this really right this time? You were doing something if some array had >0 length at some point, I don't see that happening anymore...